docs/linter-usage.md

```diff
@@ -5,10 +5,10 @@
 In order to get reports for all files in repository, run the following:
 
 ```sh
-$ noverify -cache-dir=$HOME/tmp/cache/noverify /path/to/your/project/root
+$ noverify check [-cache-dir=$HOME/tmp/cache/noverify] /path/to/your/project/root
 ```
 
-Cache dir is optional, but recommended. Next launch would be much faster with cache if you specify some cache directory.
+The cache directory is optional, by default it is already set to `temporary_folder_path/noverify`, but if you want to change the folder for storing the cache, then use the `-cache-dir` flag.
```

Comment:
I would write something like `$TMPDIR/noverify`. Otherwise it looks like `temporary_folder_path` is some kind of the literal path.